### DISCUSSION TITLE
The browser is slower when loading YouTube.

### DISCUSSION THREAD

**Discussion by franzhuang** [Jelmerro/Vieb#490] [General]
With Newly released  10.0.0. AppImage  version and binary version.
On Ubuntu based  distro.

Hi.
I found the pages loading speed slower than previous version when I maximize or minimize the window, or drag the timeline. And it becomes a little laggy when you scroll down to view comments.

**Comment by Jelmerro** [comment]
I would highly suspect this is an Electron/Chromium bug, since the performance tab is almost entirely filled with re-rendering, for both youtube and other sites with many dom nodes:
![image](https://github.com/Jelmerro/Vieb/assets/1696674/66917310-e321-4271-9c72-65c39ae23130)
I'll let you know when I know more, but according to the performance measures, Vieb's code is idling at these times of slow rendering.

**Comment by franzhuang** [comment]
There is also another situation, from 10.0.0 to 10.1.1, where when I search for 'bobcat' (for example) on Google, and then click on the 'Images' tab to go to the image search results, then I click on one of the images, sometimes the page turns red and crashes.

**Comment by franzhuang** [comment]
With 10.4.0 and different Electron/Chromium version, I think the problems I mentioned before were solved. Thanks.
